packages/playground-examples/copy/ko/JavaScript/Modern JavaScript/Import Export.ts

```diff
@@ -0,0 +1,109 @@
+//// { order: 1, target: "ES5" }
+
+// JavaScript는 2016년에 import/export를 다시 추가하였고
+// TypeScript는 파일 간의 연결과 외부 모듈로의 연결에 관한
+// 이 스타일의 지원을 마쳤습니다.
+// TypeScript는 코드와 함꼐 전달될 타입들을 허용함으로써
+// 이 문법을 확장시킵니다.
+
+// 모듈에서 코드를 import해오는 것을 봅시다.
+
+import { danger, message, warn, DangerDSLType } from "danger";
+
+// danger라는 node 모듈에서 일련의 이름 붙은 import를 가져옵니다.
+// 4개 이상의 import를 할 수 있지만,
+// 이것들이 우리가 import하기로 선택한 것들입니다.
+
+// 여러분이 무엇을 import하는지 이름을 구체적으로 적으면
+// 앱에서 사용되지 않는 코드를 제거할 수 있으며
+// 특정 파일에서 무엇이 사용되는지
+// 쉽게 이해할 수 있습니다.
+
+// 이 경우: danger, message, warn은 JavaScript import입니다.
+// 반면 DangerDSLType은 인터페이스 타입입니다.
+
+// TypeScript는 JSDoc을 이용하여 개발자의 코드를 문서화시켜주며,
+// 문서들은 또 import됩니다.
+// 예를 들어 만약 아래의 서로 다른 부분들에 커서를 올리면,
+// 각각에 대한 설명을 볼 수 있습니다.
+
+danger.git.modified_files;
+
+// 이러한 문서 표기를 제공하는 방법을 알고 싶다면
+// example:jsdoc-support를 참고하세요.
+
+// 코드를 import하는 다른 방식은
+// 모듈의 default export를 사용하는 것입니다.
+// 이 예제는 모듈 디버깅인데, 로그 기능을 하는 함수를 내보냅니다.
```

Comment:
```suggestion
// 이 예제는 디버그 모듈인데, 로그 기능을 하는 함수를 내보냅니다.
```
모듈 디버깅은 의미가 조금 다른 것 같아요.